bin/standardize

```diff
@@ -2,4 +2,4 @@
 
 bundle exec magic_frozen_string_literal
 bundle exec standardrb --fix
-cd ./javascript && yarn run prettier-standard cable_ready.js
+yarn run prettier-standard ./javascript/*.js
```

Comment:
In case you get any nested files in the future, you may want this to be:

./javascript/**/*.js